docker-registry-sync/Dockerfile

```diff
@@ -1,13 +1,17 @@
-# based on https://github.com/xelalexv/dregsy/blob/0.5.1/hack/dregsy.alpine.Dockerfile
-FROM xelalex/dregsy:0.5.1-alpine@sha256:1adb89b1310128367a55dbbe6b0656c36558986e893fa96e610303069d966578
+FROM python:3.11-alpine
```

Comment:
you are aware that alpine is not very recommended? it behaves a bit strangely with many things due to the musl backend.

Reply:
Yes I am aware it is not perfect. But I have not seen any issue with it so far.